### ISSUE TITLE
Flexural reticulate hyperpigmentation

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8020]
**Preferred term label:**

**Synonyms**
Reticulate Hyperpigmentation of the Flexures 
reticular pigmented anomaly of the flexures

**Definition (free text, please give PubMed ID)**
Hyperpigmented macules with a reticulate (net-like) pattern involving the neck, axillae, groins, and other flexures like the inframammary folds in females. Histopathology shows orthokeratosis or hyperkeratosis, thinning of the suprapapillary epithelium, elongation of the rete ridges, hyperpigmentation of the basal layer, and follicular cysts. A perivascular lymphohistiocytic infiltrate can be observed in the papillary dermis.
PMID:24470653

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
612079 
PMID: 18439547  no frequency

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000745
- label: Flexural reticulate hyperpigmentation
- parents: [Reticulated skin pigmentation (HP:0007427)](https://hpo.jax.org/app/browse/term/HP:0007427)
- pmid: PMID:24470653
- def: Hyperpigmented macules with a reticulate (net-like) pattern involving the neck, axillae, groins, and other flexures like the inframammary folds in females. Histopathology shows orthokeratosis or hyperkeratosis, thinning of the suprapapillary epithelium, elongation of the rete ridges, hyperpigmentation of the basal layer, and follicular cysts. A perivascular lymphohistiocytic infiltrate can be observed in the papillary dermis.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8020
- synonyms: Reticular pigmented anomaly of the flexures(NONE)|Reticulate hyperpigmentation of the flexures(NONE)
